### ISSUE TITLE
Used to be able to just download a EXE togehter with the used folders. 

### ISSUE THREAD

**Issue by ThePoi** [DontTalkToMeThx/BooruSlideshowElectron#13]
So i just formatted my whole system and I used to have a booruslideshow folder with a .exe file. I followed the instructions and ran "npm run build-win64" and got a .exe. But it doesn't work. the search does nothing. And I remember that I could download the pre-built program from some github page. I hate the firefox extension because if I click the video it doesn't open the source link, it just pauses the video.

So my question is where do I find a pre-built version that works and opens images/videos to their respective sources in my browser. 

I understand that the creator of this Electron version might not want to work on the project but it's crazy that the only option to use this app is to download node and build it my self. How freaking hard is it to just provide a working .exe in the releases?

**Comment by ThePoi** [comment]
Ignore the whole text that I wrote. If you look in the branches there is a compiled version that works perfectly. Sorry for forgetting to look there first. I spend a week trying to get this thing to work and the branch works out of the box.

**Comment by ThePoi** [comment]
Ignore the whole text that I wrote. If you look in the branches there is a compiled version that works perfectly. Sorry for forgetting to look there first. I spend a week trying to get this thing to work and the branch works out of the box.

**Comment by searhx** [comment]
it's not an updated version though..

**Comment by ThePoi** [comment]
> it's not an updated version though..

Yeah and that’s pretty sad. I actually tried and managed to compile and get a complete folder with exe. But the whole program looked wrong and nothing worked. I’m not used to compiling stuff myself so I probably did something wrong.

**Comment by searhx** [comment]
me and a friend both tried to compile it, and we both were unable to use the search function
